.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,67 @@
+name: Build and publish scroll-alpine image
+# This workflow aims to build images from the feat-deterministic-deployment branch
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Clone, Build, Publish
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Update submodules recursively
+        run: git submodule update --init --recursive
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
+        with:
+          node-version: '21'
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Dockerhub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build deploy image
+        id: build_deploy_image
+        env:
+          REGISTRY: ${{ steps.login-ecr.outputs.registry }}
```

Comment:
Why do we have ECR here but Docker Hub above, is this correct?

Reply:
it's because we deploy 2 times, once to ECR and once to dockerhub.